src/rsa.rs

```diff
@@ -289,6 +316,31 @@ impl RSAKeyPair {
     pub fn public_modulus_len(&self) -> usize {
         unsafe { RSA_size(self.rsa.as_ref()) }
     }
+
+    /// Pad the message `msg` with the padding algorithm `padding_alg` and write
+    /// the resulting signature into `signature`. The random number generattor `rng`
+    /// is used for blinding (to help make signing constant time).
```

Comment:
The `(to make signing constant time)` is incorrect; just drop that parenthetical. (The blinding is actually used because the rest of RSA isn't 100% constant time, though it tries hard to be, and so the blinding randomizes the timing.)